src/main/java/com/jcabi/aether/Aether.java

```diff
@@ -413,13 +413,14 @@ private Settings settings() {
     private Settings invokers(final SettingsBuilder builder,
         final SettingsBuildingResult result) {
         Settings main = result.getEffectiveSettings();
-        final Path path = Paths.get(
-            System.getProperty("user.dir"), "..", "interpolated-settings.xml"
+        final File[] files = new File(System.getProperty("user.dir"))
+            .getParentFile().listFiles(
+                (FileFilter) new NameFileFilter("interpolated-settings.xml")
```

Comment:
Yes, because there are 2 implementations of '''listFiles''' that accepts 2 interfaces, but '''NameFileFilter''' class implements both of them.